Mathlib/LinearAlgebra/FreeProduct/Basic.lean

```diff
@@ -133,19 +141,62 @@ of tensor powers are (noncomputably) equivalent as `R`-algebras. -/
 /-- The generating equivalence relation for elements of the free tensor algebra
 that are identified in the free product -/
 inductive rel : FreeTensorAlgebra R A → FreeTensorAlgebra R A → Prop
+  /--`rel` relates the image of any `(1 : A i)` to `1`.-/
```

Comment:
```suggestion
  /-- `rel` relates the image of any `(1 : A i)` to `1`. -/
```
Same below

@adomani, the docstring linter didn't catch this! 👮

Reply:
Wow, interesting!  Let me investigate!

Reply:
Ok, the reason is that the doc-string linter only inspects doc-strings contained in `declModifiers`.  This is mostly to avoid looking at `#adaptation_note`.  However, I agree that inspecting doc-strings of structure fields is also useful.  Let me see how to extend the linter!

Reply:
#24673.  Let's see what else it catches!